### DISCUSSION TITLE
Bulgarian Language Leader

### DISCUSSION THREAD

**Discussion by TUISTERa** [OHF-Voice/intents#426] [Language Leaders]
Hi! I want to help with Bulgarian language.

**Comment by thebgrian** [comment]
I can also help with Bulgarian language translations and help with the team.

**Comment by synesthesiam** [comment]
Thanks! I've added empty Bulgarian templates (`bg`) to the repository. Please start with `_common.yaml` and the `homeassistant*.yaml` files in `sentences/bg`

**Comment by Xerrathbg** [comment]
I want to help as well!

**Comment by hristo-atanasov** [comment]
I want to help too .. I also added Google TTS bulgarian language support and also I did it in HA.

**Comment by SectoidBG** [comment]
I want to help!

**Comment by synesthesiam** [comment]
Thanks to everyone for volunteering! Is anyone interested in being a language leader?
There are Bulgarian templates in repo, so we're ready for contributions :+1:

**Reply by hristo-atanasov** [reply]
@synesthesiam  Ok, I just made my initial commit and first PR. We can discuss here if it is OK or it needs some changes, as I'm not 100% sure if I've got the instructions how to populate it. This way everyone who wants to contribute can see the "needs correction log".

My PR: [https://github.com/home-assistant/intents/pull/438](https://github.com/home-assistant/intents/pull/438)

Also I can be language leader too, as I see no one interested yet.

**Reply by TUISTERa** [reply]
I am OK for @hristo-atanasov to be the language leader, as i am busy at the moment , I want and I will help , but I dont have the time for managing too

**Comment by LZ7AA** [comment]
Good evening guys.
My name is Kiril Sokolov, I come with IT and project management background, although not a developer (but I know some stuff). I am a big fan of HA and I'm interested and willing to help with in voice controls.
My timetable is relaxed these days and if you need assistance with coordination and administration I can jump in and help.

**Comment by BossNeo** [comment]
Hello i am so happy to see people here :) 
But sadly nothing much is done and i see a a lot off mistakes in the files that is alredy upload. Guys please take that serius, dont rush just to show off.  I will going to help too,  i am IT too with with C code knowedges and been with HA from 1,5 year already and i realy want to helpon that voice control. Lets make it good.

**Comment by synesthesiam** [comment]
Awesome! Adding @hristo-atanasov as language leader now :+1:

**Comment by synesthesiam** [comment]
@hristo-atanasov Can you DM me on Discord? synesthesiam#5707

**Reply by hristo-atanasov** [reply]
Yes, I can. In one hour, I'm finishing with work and I will.

**Comment by dimosgit** [comment]
Hello everyone I would gladly contribute especially to the Voice Preview. Is there a voice leader already ? @hristo-atanasov are you still the voice leader ?

**Reply by hristo-atanasov** [reply]
Yes, I am. Actually I'm the language leader.

**Reply by dimosgit** [reply]
Здравей, Христо :) Това е чудесно. Имате ли нужда от повече contributions ? Защото видях, че voice nabu, все още не поддържа български. @hristo-atanasov

**Reply by hristo-atanasov** [reply]
Здравейте! Не отказвам контрибуции. :)  Помощ винаги е добре дошла. Има все още много непреведени изречения, тестове и отговори.
